src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js

```diff
@@ -26,6 +26,8 @@ export function RedirectCreator(props) {
         code: code === 1 ? 301 : 302 // API expects a 301/302 value
       })
     );
```

Comment:
Assuming the `createRedirect` is returning the fetch promise this should be a thenable value. Lets put the reset into a `then` clause. As well as lets ensure a system notification is being shown on when a bad request or error occurs. This way on a bad request we don't wipe the fields and a user can make a correction to try again.

Reply:
add then(), added notify success to the store for successful redirects.